packages/homepage/src/css/global.css

```diff
@@ -12,7 +12,7 @@ body {
   color: white;
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
     Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue';
-  /* overflow-x: hidden; */
+  overflow-x: hidden;
```

Comment:
Maybe uncommenting this line is not a big deal, or maybe this line was commented on for some unknown reason. I'd try to apply this in the component that has the problem and doesn't globally. As I could see, the following section is bigger than the window width in its initial state, but after scrolling, it fits into the window.
<img width="1655" alt="Screenshot 2021-08-31 at 09 11 29" src="https://user-images.githubusercontent.com/4838076/131467018-27c9b171-93db-4229-bde6-f2d6379f326d.png">